geopandas/io/tests/test_file.py

```diff
@@ -576,6 +576,13 @@ def test_read_file_remote_zipfile_url(engine):
     assert isinstance(gdf, geopandas.GeoDataFrame)
 
 
+@pytest.mark.web
+def test_read_file_remote_zipfile_url_without_extension(engine):
+    url = "https://geonode.goosocean.org/download/480"
```

Comment:
@ocefpaf do you know if this is a stable url that we could use for testing?

Reply:
Otherwise I suppose I could upload a small zip file without extension to our github repo. That's maybe better

Reply:
Stable enough and you could use pytest-vcr too. But adding it to GH is probably better.